src/_layouts/base.html

```diff
@@ -39,9 +39,9 @@
 <meta property="og:type" content="{% block og_type %}website{% endblock %}">
 <meta property="og:url" content="{{ request.url }}">
 <meta property="og:image"
-      content="{{ request.scheme }}://{{ request.get_host() }}{{ static('img/logo_open-graph_facebook.png') }}">
+      content="http://files.consumerfinance.gov/f/images/ask-cfpb-social-mortgages.original.png">
```

Comment:
should these be https?

Reply:
Unfortunately, no - https can't resolve them.

Reply:
try: https://s3.amazonaws.com/files.consumerfinance.gov/f/images/ask-cfpb-social-mortgages.original.png

Reply:
Will do!